doc/source/flexibility/innodb_fts_improvements.rst

```diff
@@ -0,0 +1,48 @@
+.. _innodb_fts_improvements:
+
+====================================
+InnoDB Full-Text Search improvements
+====================================
+
+By default n-gram indexes check the `stopwords list
+<https://dev.mysql.com/doc/refman/5.7/en/fulltext-stopwords.html>`_,
+to see if any indexed elements *contain* one of the words on that list.
+
+Using this list for n-gram indexes isn't always suitable, as an example, any
+item that contains ``a`` or ``i`` will be ignored. Another word that can't be
+searched is ``east``, this one will find no matches because ``ea`` is on the
```

Comment:
Although this is exactly what is stated in the blueprint document `ea` is not a default stopword.
I would suggest to change this to just `a`.